src/test/kotlin/no/nav/henvendelsesarkiv/ComponentSpec.kt

```diff
@@ -39,9 +39,9 @@ object ComponentSpec: Spek({
             }
```

Comment:
Kaller man tester Spec i Kotlin?

Reply:
Spek er et alternativ til JUnit, der man sier at unit tester i praksis er dokumentasjon (spesifikasjon) av kode. Derfor prøvde jeg det for å se hva dere synes :)

Reply:
Åja, kult :)